hudi-client/hudi-client-common/src/main/java/org/apache/hudi/config/HoodieIndexConfig.java

```diff
@@ -57,7 +64,11 @@ public class HoodieIndexConfig extends HoodieConfig {
 
   public static final ConfigProperty<String> INDEX_TYPE = ConfigProperty
       .key("hoodie.index.type")
-      .noDefaultValue()
+      .defaultValue(BLOOM.name())
```

Comment:
can we make it `SIMPLE` here? we switched to SIMPLE default here https://github.com/apache/hudi/pull/4659

Reply:
fixed